template/Dockerfile.debian

```
@@ -1,3 +1,4 @@
+{{ include "template-helper-functions" }}
```

Need the `-` to tell it to slurp up the whitespace (so that it doesn't generate a blank line in the final Dockerfiles.

```suggestion
{{ include "template-helper-functions" -}}
```